1-the-basics/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  font-family: "Inter", sans-serif;
+  font-size: 1rem;
+  font-weight: 600;
+  color: rgb(0, 65, 73);
```

Comment:
get use to sorting your CSS properties from A-Z

Reply:
I usually sort the properties according to their functionality:
1. Positioning and Flow.
2. size and box properties
3. color and shape
4. animations or transitions
does this depend on the project I'm in or is it the developer's criteria?